native/src/modules/app/StateType.js

```diff
@@ -59,18 +63,22 @@ export type PoiRouteConfigType = {|
 |}
 
 export type PoiRouteStateType = {|
+  +routeType: 'poi',
```

Comment:
Wouldn't is be nicer to add routeType to EventRouteConfigType? 
Tested it, flow is also happy with it.

Reply:
Hmm, to me it feels inconsistent to only add it to the poirouteconfig and/or eventrouteconfig. A if I also introduce the routeType to the CategoryRouteConfig it leads to another refactoring task as this object is reused in some of the ActionTypes. Imo it's better to reduce coupling here and keep it as it is

Reply:
Yeah you are right. Thanks for explation.